gulpfile.coffee

```diff
@@ -7,24 +8,27 @@ coffee = require('gulp-coffee')
 runSequence = require('run-sequence')
 packageJSON = require('./package.json')
 
+{ loadEnv } = require('./tests/util')
+
 OPTIONS =
 	config:
 		coffeelint: path.join(__dirname, 'coffeelint.json')
 	files:
 		coffee: [ 'lib/**/*.coffee', 'tests/**/*.coffee', 'gulpfile.coffee' ]
 		app: 'lib/**/*.coffee'
-		integration: 'tests/integration.coffee'
+		integration: 'tests/integration.spec.coffee'
 		javascript: 'build/**/*.js'
 	directories:
 		doc: 'doc/'
 		build: 'build/'
 
 gulp.task 'coffee', ->
 	gulp.src(OPTIONS.files.app)
-		.pipe(coffee()).on('error', gutil.log)
+		.pipe(coffee(header: true)).on('error', gutil.log)
```

Comment:
@jviotti it'd be awesome to gradually add this to our other modules as well (as changes are made to them), it lets the .js files get suppressed in the diff and is overall so much easier to review :D